bindings/rust/s2n-tls/Cargo.toml

```diff
@@ -15,6 +15,7 @@ unstable-ktls = ["s2n-tls-sys/unstable-ktls"]
 quic = ["s2n-tls-sys/quic"]
 fips = ["s2n-tls-sys/fips"]
 pq = ["s2n-tls-sys/pq"]
+testing = []
```

Comment:
we should name this `unstable-testing` so it's clear we don't guarantee backward-compat